facebook/docusaurus
# feat(v2): allow skipping build docs for next version

## Motivation

Resolve #2813

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

1. In the config file for `docs` set the `excludeNextVersionDocs` field with the `true` value.
1. Try running dev server running `yarn start` and make sure that docs are accessible by `docs/next` path
1. Try building the site running `yarn build` and make sure `build/docs/next` dir does not exist. 
4. Remove `excludeNextVersionDocs` filed and build site again, and make sure `build/docs/next` dir appears. 

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)
